{"totalCount":3,"grade":"C","repositories":[{"url":"https://github.com/MTES-MCT/apilos","vulnerabilityAlerts":{"totalCount":3,"nodes":[{"dismissedAt":null,"createdAt":"2025-03-06T04:48:45Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"Jinja2"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-cpwx-vrp4-4pq7"},{"type":"CVE","value":"CVE-2025-27516"}],"references":[{"url":"https://github.com/pallets/jinja/security/advisories/GHSA-cpwx-vrp4-4pq7"},{"url":"https://github.com/pallets/jinja/commit/90457bbf33b8662926ae65cdde4c4c32e756e403"},{"url":"https://nvd.nist.gov/vuln/detail/CVE-2025-27516"},{"url":"https://github.com/advisories/GHSA-cpwx-vrp4-4pq7"}]}}},{"dismissedAt":null,"createdAt":"2025-03-06T22:59:46Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"Django"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-p3fp-8748-vqfq"},{"type":"CVE","value":"CVE-2025-26699"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2025-26699"},{"url":"https://docs.djangoproject.com/en/dev/releases/security"},{"url":"https://groups.google.com/g/django-announce"},{"url":"https://www.djangoproject.com/weblog/2025/mar/06/security-releases"},{"url":"http://www.openwall.com/lists/oss-security/2025/03/06/12"},{"url":"https://github.com/advisories/GHSA-p3fp-8748-vqfq"}]}}},{"dismissedAt":null,"createdAt":"2025-03-06T22:59:46Z","securityVulnerability":{"severity":"MODERATE","package":{"name":"Django"},"advisory":{"identifiers":[{"type":"GHSA","value":"GHSA-p3fp-8748-vqfq"},{"type":"CVE","value":"CVE-2025-26699"}],"references":[{"url":"https://nvd.nist.gov/vuln/detail/CVE-2025-26699"},{"url":"https://docs.djangoproject.com/en/dev/releases/security"},{"url":"https://groups.google.com/g/django-announce"},{"url":"https://www.djangoproject.com/weblog/2025/mar/06/security-releases"},{"url":"http://www.openwall.com/lists/oss-security/2025/03/06/12"},{"url":"https://github.com/advisories/GHSA-p3fp-8748-vqfq"}]}}}]},"grade":"C"}]}